docs/examples/safe-remote.rst

```diff
@@ -41,26 +41,35 @@ you can use state machine-like constructs inside a contract.
             _;
         }
 
+        /// Only the buyer can call this function.
+        error OnylBuyer();
+        /// Only the seller can call this function.
+        error OnylBuyer();
+        /// The function cannot be called at the current state.
+        error InvalidState();
+        /// The provided value has to be even.
+        error ValueNotEven();
+
         modifier onlyBuyer() {
             require(
                 msg.sender == buyer,
-                "Only buyer can call this."
+                OnlyBuyer()
             );
```

Comment:
After removing strings this is pretty short so I'd make these modifiers more compact:
```solidity
     require(msg.sender == buyer, OnlyBuyer());
```